lib/repository.js

```diff
@@ -55,8 +55,8 @@ const fetchFiles = (repoName, files) => {
     }
 
     files.forEach((file) => {
-        const downloadUrl = `https://raw.githubusercontent.com/${repoName}/master/${file}`;
-        sync(downloadUrl, file);
+        // @TODO - parse tag or branch name from `dependencies`
+        sync(repoName, 'master', file);
```

Comment:
Yup 😄 Didn't think you liked that pattern. I don't see implicit returns anywhere.

Thanks for invite btw 😀